azext_iot/central/params.py

```diff
@@ -148,3 +148,11 @@ def load_central_arguments(self, _):
             help="Maximum number of messages to recieve from target device before terminating connection."
             "Use 0 for infinity.",
         )
+
+    with self.argument_context("iot central dps-keygen") as context:
+
+        context.argument(
+            "masterkey",
+            options_list=["--masterkey", "-k"],
+            help="Masterkey - Primary group SAS token to generate device keys",
```

Comment:
In the interest of being more inclusive I've heard things like Microsoft is moving away from "master" in their terminology. We should update this to be something like "basekey", "rootkey", "trustedkey", or "primarykey" or similar

Reply:
This will also be more consistent with our docs terminology, since we don't seem to use the word master there at all

![image](https://user-images.githubusercontent.com/42591144/90432505-7b296400-e07f-11ea-8579-994ef7804723.png)